### PR TITLE
fix: fleet validation changes

### PR DESCRIPTION
**Pull request by nameetjambla**
In these files, I have changes the validation messages and pattern as suggested by Param Paji.

### CODE REVIEW

File: src/app/pages/fleet/drivers/add-driver/add-driver.component.html [hunk 1]
```diff
@@ -1667,7 +1667,7 @@ <h5 class="modal-title">Add Group</h5>
                       Group name is required.
                     </div>
                     <div *ngIf="groupModel.errors.pattern">
-                        Group name must contain letters.
+                        Group name must contain numbers.
```
Comment (FH-sandeepkaur1) [inline src/app/pages/fleet/drivers/add-driver/add-driver.component.html:1670]: Please change msg - Group name can only contain alphanumeric characters

File: src/app/pages/fleet/reminders/service-reminder/add-reminder/add-reminder.component.html [hunk 2]
```diff
@@ -244,13 +244,13 @@ <h5 class="modal-title" id="serviceTaskModalTitle">Add Service Task</h5>
...
                      <input [(ngModel)]="serviceTask.taskName" name="taskName" type="text" class="form-control"
-                        placeholder="Enter service task name" #txtTask="ngModel" required pattern="^[a-zA-Z\s]+">
+                        placeholder="Enter service task name" #txtTask="ngModel" required pattern="^[a-zA-Z0-9\s]+">
                         <div *ngIf="txtTask.invalid && (txtTask.dirty || txtTask.touched)" class="text-danger">
                            <div *ngIf="txtTask.errors.required">
                              Service task name is required.
                            </div>     
                            <div *ngIf="txtTask.errors.pattern">
-                                  Service task name must contain letters.
+                                  Service task name must alphanumeric characters.
```
Comment (FH-sandeepkaur1) [inline src/app/pages/fleet/reminders/service-reminder/add-reminder/add-reminder.component.html:253]: please check message

**Review by FH-sandeepkaur1 — COMMENTED** [review]
Please address comments

**Review by kunalfleethawks — APPROVED** [review]

File: src/app/pages/fleet/reminders/service-reminder/add-reminder/add-reminder.component.html (hunk 2 above)
Comment (nameetjambla) [reply src/app/pages/fleet/reminders/service-reminder/add-reminder/add-reminder.component.html:253]: done

File: src/app/pages/fleet/drivers/add-driver/add-driver.component.html (hunk 1 above)
Comment (nameetjambla) [reply src/app/pages/fleet/drivers/add-driver/add-driver.component.html:1670]: done

**Pull request merged**